nncf/onnx/hardware/fused_patterns.py

```diff
@@ -372,6 +459,16 @@ def create_linear_arithmetic_activations() -> GraphPattern:
     return linear
 
 
+@ONNX_HW_FUSED_PATTERNS.register(HWFusedPatternNames.LINEAR_ARITHMETIC_ARITHMETIC)
+def create_linear_arithmetic_arithmetic() -> GraphPattern:
```

Comment:
I think it could be `linear_shift_scale` and we can use `create_shift_scale()`

Reply:
Good idea! Done